lib/phoenix_live_view/html_tokenizer.ex

```diff
@@ -457,4 +480,13 @@ defmodule Phoenix.LiveView.HTMLTokenizer do
   defp pop_brace(%{braces: [pos | braces]} = state) do
     {pos, %{state | braces: braces}}
   end
+
+  defp downcase?(<<c, _::binary>>) when c in ?A..?Z, do: false
+  defp downcase?(<<_, rest::binary>>), do: downcase?(rest)
+  defp downcase?(<<>>), do: true
+
+  defp warn(message, file, line) do
+    stacktrace = Macro.Env.stacktrace(%{__ENV__ | file: file, line: line})
```

Comment:
I think we want:

```suggestion
    stacktrace = Macro.Env.stacktrace(%{__ENV__ | file: file, line: line, module: nil})
```

Since we don't want to point to this module+function.